### ISSUE TITLE
[experiment] Same contrasts included in training / validation / testing

### ISSUE THREAD

**Issue by charleygros** [ivadomed/ivadomed#30]
Protocol: Do the inference with parameters included in the training (as opposed to asking FILM to extrapolate on new parameter combination). FiLMed-Unet versus Unet.

When posting new results for this experiment, please provide:
- commit ID that you are using
- `config_file.json`
- Tensorflow learning curves (ie training & validation loss)
- evaluation metrics on the testing dataset
